console.log("hello");
console.log("hi");
let a= 10;
try {
    console.log(a);
}
catch {
    console.log("caught a error ");
}
console.log("hello2");
console.log("hi2");